src/main/java/io/quarkus/gizmo2/impl/constant/ConstantImpl.java

```diff
@@ -72,8 +81,6 @@ public static ConstantImpl of(DynamicConstantDesc<?> dcd) {
             return of(desc);
         } else if (dcd instanceof VarHandle.VarHandleDesc desc) {
             return of(desc);
-        } else if (dcd.bootstrapMethod().equals(ConstantDescs.BSM_NULL_CONSTANT)) {
-            return ofNull(dcd.constantType());
```

Comment:
What happened here?

Reply:
This is duplicated a few lines below, with the comment `"wrong spelling" of null constant`. I figured one case is enough :-)